pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -81,6 +81,10 @@ func validateKubeSchedulerProfile(path *field.Path, profile *config.KubeSchedule
 	if len(profile.SchedulerName) == 0 {
 		allErrs = append(allErrs, field.Required(path.Child("schedulerName"), ""))
 	}
+	if profile.PercentageOfNodesToScore < 0 || profile.PercentageOfNodesToScore > 100 {
+		allErrs = append(allErrs, field.Invalid(path.Child("percentageOfNodesToScore"),
```

Comment:
Please make it UTed:

Cases w/ or w/o top-level percentageOfNodesToScore, multiply the cases w or w/o  profile-level one.